packages/expo-splash-screen/src/SplashScreen.ts

```diff
@@ -2,25 +2,9 @@ import { UnavailabilityError } from '@unimodules/core';
 
 import ExpoSplashScreen from './ExpoSplashScreen';
 
+// @needsAudit
 /**
- * Makes the native splash screen stay visible until `SplashScreen.hideAsync()` is called.
- * It has to be called before any view is rendered.
- *
- * @example
- * ```typescript
- * // top level component
- *
- * SplashScreen.preventAutoHideAsync()
- *  .then(result => console.log(`SplashScreen.preventAutoHideAsync() succeeded: ${result}`))
- *  .catch(console.warn); // it's good to explicitly catch and inspect any error
- *
- * class App extends React.Component {
- *   ...
- *   // Hide SplashScreen once your app content is ready to be displayed.
- *   await SplashScreen.hideAsync()
- *   ...
- * }
- * ```
```

Comment:
Don't we want to include `@example` tag inside tsdocs? 🤔

Reply:
https://github.com/expo/expo/pull/12367#discussion_r605274762

Reply:
Also it looks like this code is a simplified version of example provided in Usage section on the doc page.